MACOS_USAGE_GUIDE.md

```diff
@@ -0,0 +1,73 @@
+# Usage Guide
+
+A Step-by-Step guide on how to embed a Python interpreter in a MacOS app (The process for an iOS app should be quite similar).
+No need to delete the Sandbox (you need it to be able to submit your MacOS App to the App Store).
+No need to `Disable Library Validation`.
+
+1. Add PythonKit SPM:
+https://github.com/pvieito/PythonKit
+
+2. Download Released framework for the desired Python version (for MacOS platform):
+https://github.com/beeware/Python-Apple-support
+
+3. Extract the `python-stdlib` and `Python.xcframework` from the `tag.gz` archive.
+
+4. Copy `python-stdlib` and `Python.xcframework` to the root of the MacOS App, preferably via Xcode.
+
+5. Xcode General -> Frameworks:
+	5.1. Should already be there:
+		- `Python.xcframework` is set as `Do Not Embed`
+		- `PythonKit`
+	5.2. Add additional required framework:
+		- `SystemConfiguration.framework` set as `Do Not Embed`
+
+6. Xcode `Build Phases`:
+	6.1. Verify `Copy Bundle Resources` contains `python-stdlib`.
+	6.2. Add bash script to Sign `.so` binaries in `python-stdlib/lib-dynload/`:
+	IMPORTANT NOTE: `.so` binaries must be signed with your TeamID, if you need to use `Sign and Run Locally` it will be signed as ad-hoc, and you will need to `Disable Library Validation`.
+```bash
+set -e
+echo "Signing as $EXPANDED_CODE_SIGN_IDENTITY_NAME ($EXPANDED_CODE_SIGN_IDENTITY)"
+find "$CODESIGNING_FOLDER_PATH/Contents/Resources/python-stdlib/lib-dynload" -name "*.so" -exec /usr/bin/codesign --force --sign "$EXPANDED_CODE_SIGN_IDENTITY" -o runtime --timestamp=none --preserve-metadata=identifier,entitlements,flags --generate-entitlement-der {} \;
+```
+
+7. Create a file called `module.modulemap` with the following code:
+```
+module Python {
```

Comment:
It seems like it would be easier to include this modulemap file in the actual support package; AFAICT, it should be a fairly straightforward thing to add as an extra file in the `patch/Python` folder that gets copied into the final release package, either as part of the `PYTHON_XCFRAMEWORK-$(os)` target, or just prior to building the tarball in the `dist/Python-$(PYTHON_VER)-$(os)-support.$(BUILD_NUMBER).tar.gz` target.